presets.go

```diff
@@ -106,8 +106,11 @@ var PRESETS = map[string]QueryPreset{
 	"iran": QueryPreset{
 		include: []string{"iran", "tehran", "mashhad", "isfahan", "esfahan", "karaj", "shiraz", "tabriz", "qom", "ahvaz", "ahwaz", "kermanshah", "urmia", "rasht", "kerman"},
 	},
-	"congo": QueryPreset{
-		include: []string{"congo", "drc", "kinshasa", "lubumbashi", "bukavu", "kananga", "goma"},
+	"congo Kinshasa": QueryPreset{
+		include: []string{"congo+kinshasa","drc", "cod", "kinshasa", "lubumbashi", "bukavu", "kananga", "goma", "mbuji+mayi", "likasi", "kolwezi","kalemie","uvira","matadi","moba","Kamina","Kabalo","kungurume"},
+	},
+	"congo brazzaville": QueryPreset{
+		include: []string{"congo+brazza", "congo+brazza", "cog", "brazzaville", "djambala", "pointe-noire", "sibiti", "owando", "madingou", "loango", "kinkala", "impfondo", "dolisie"},
```

Comment:
```suggestion
		include: []string{"congo+brazza", "cog", "brazzaville", "djambala", "pointe+noire", "sibiti", "owando", "madingou", "loango", "kinkala", "impfondo", "dolisie"},
```
removed repeated term + switched to using `pointe noire` instead of `pointe-noire` (will match more results, hopefully!)